src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ActivityExtensions.cs

```diff
@@ -129,25 +129,25 @@ internal static OtlpTrace.Span ToOtlpSpan(this Activity activity)
             return otlpSpan;
         }
 
-        private static OtlpTrace.Status ToOtlpStatus(Status status)
-        {
-            if (!status.IsValid)
-            {
-                return null;
-            }
-
-            var otlpStatus = new Opentelemetry.Proto.Trace.V1.Status
-            {
-                // The numerical values of the two enumerations match, a simple cast is enough.
-                Code = (OtlpTrace.Status.Types.StatusCode)status.CanonicalCode,
-            };
-
-            if (!string.IsNullOrEmpty(status.Description))
-            {
-                otlpStatus.Message = status.Description;
-            }
-
-            return otlpStatus;
+        private static OtlpTrace.Status ToOtlpStatus(Status status)
```

Comment:
This file got the CRLF and LF mixed up, just replaced CRLF with LF with no other changes.